applications/nrf5340_audio/src/audio_module/src/amod_api.h

```diff
@@ -0,0 +1,492 @@
+/*
+ * Copyright(c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#ifndef _AMOD_API_H_
+#define _AMOD_API_H_
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <zephyr/kernel.h>
+#include "data_fifo.h"
+#include "aobj_api.h"
+
+/**
+ * @brief Modules private handle.
```

Comment:
Hmm is it allowed to write "Modules" here? Or is it "Module's"?

This is relevant several places in this file

Reply:
Should be Module's as it is the possession of the module not multiple modules.